resources/install.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ' | /opt/bin/base64)
 sed -i s/cGFzc3dvcmQtZ29lcy1oZXJlCg==/$password/g /var/lib/gravity/resources/grafana.yaml
-for filename in security smtp influxdb grafana heapster kapacitor telegraf alerts
+for filename in security smtp influxdb grafana heapster kapacitor telegraf alerts kube-state-metrics
 do
     /opt/bin/kubectl create -f /var/lib/gravity/resources/${filename}.yaml
 done
+
+if [ $(/opt/bin/kubectl get nodes -l gravitational.io/k8s-role=master -o name | wc -l) -ge 3 ]
```

Comment:
Why not just launch it as a daemon set with node selector on masters? Because this applies only to install, any upgrade will invalidate this (unless it does the same, but it's messy).